src/bioregistry/data/bioregistry.json

```diff
@@ -108,7 +108,11 @@
     "example": "4DNESWX1J3QU",
     "homepage": "https://data.4dnucleome.org/experiment-set-replicates/",
     "name": "4D Nucleome Data Portal Experiment Replicate",
-    "uri_format": "https://data.4dnucleome.org/experiment-set-replicates/$1"
+    "uri_format": "https://data.4dnucleome.org/experiment-set-replicates/$1",
+    "wikidata": {
+      "database": "Q110613797",
+      "property": "Q4117139"
```

Comment:
This is not a valid property corresponding to the database (it should also use the "prefix" key)